examples_new/optimization/scipy/optimize_power_density_1D.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 NREL
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+import matplotlib.pyplot as plt
+import floris.tools as wfct
+import floris.tools.visualization as vis
+from floris.tools.optimization.scipy.power_density_1D \
+    import PowerDensityOptimization1D
+import numpy as np
+import os
+
+# Instantiate the FLORIS object
+file_dir = os.path.dirname(os.path.abspath(__file__))
+fi = wfct.floris_interface.FlorisInterface(
+    os.path.join(file_dir, '../../example_input.json')
+)
+
+# Set turbine locations to 3 turbines in a triangle
```

Comment:
Why a triangle?

Reply:
Ah, whoops, copy and paste error from another example. Fixed!